portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -74,6 +74,34 @@ definition {
 		BlogsEntry.viewInlineImage(uploadFileName = "Document_1");
 	}
 
+	@priority = "4"
+	test ShouldBeAbleToAddALinkToCKEditor {
```

Comment:
I analyzed this further and I think `CanAddHyperLinkToText` would better describe the test.